lib/searchName.ml

```diff
@@ -207,4 +207,4 @@ let print conf base specify unknown =
         [ Sosa; Key; Surname; ApproxKey; PartialKey; DefaultSurname ]
       in
       search conf base sn order specify unknown
-  | None, None -> Hutil.incorrect_request conf
+  | None, None -> Hutil.incorrect_request conf ~comment:"Missing fn and sn"
```

Comment:
since this will be a error the user will see, maybe we should write first name and surname in full for less confusion

Reply:
ok